### PR TITLE
[codestyle] Removed redundant elses

### PR DESCRIPTION
**Pull request by ghost**
This PR Removes redundant elses that contain returns in the previous if.
/components/com_users/controllers/user.php

### CODE REVIEW

**Comment by zero-24** [comment]
Looks good thanks @nonumber <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/7718">issues.joomla.org/joomla-cms/7718</a>.</sub>

**Comment by zero-24** [comment]
As we change the logic and not only simple CS. I just remove the RTC but mark my successful test. On really codestyle fixes we don't need tests but as we change the logic here we need tests. Just that all looks good as a simple Code Review is not easy possible.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/7718">issues.joomla.org/joomla-cms/7718</a>.</sub>

**Comment by ceus1984** [comment]
I have tested this item :white_check_mark: successfully on aa7ae6495db8dc4eb3f7c5c4fb3644bbdff1c6fb<br /><br />Tested Login and Logout. Wrong password and "Rememberme" Cookie<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/7718">issues.joomla.org/joomla-cms/7718</a>.</sub>

**Comment by zero-24** [comment]
@nonumber Can you rebase here? Than i can set it RTC. Thanks

File: components/com_users/controllers/user.php [hunk 1]
```diff
@@ -264,6 +256,8 @@ public function remind()
 
 			return false;
 		}
+<<<<<<< HEAD
+=======
```
Comment (zero-24) [inline components/com_users/controllers/user.php]: @nonumber looks like not indend right?

**Comment by nonumber** [comment]
Sorry for the mucking around. Have switched from Windows to a Mac a few weeks ago. So still figuring out the new workflow for github :S

**Comment by zero-24** [comment]
I have tested this item :white_check_mark: successfully on adb44f2246a0592b28c921563dbbf6e8d087e10d<br /><br />Just tested again the rebased version. Thanks @nonumber <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/7718">issues.joomla.org/joomla-cms/7718</a>.</sub>

**Comment by zero-24** [comment]
RTC. Thanks. Mave fun with your new Mac :smiley: <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="http://issues.joomla.org/tracker/joomla-cms/7718">issues.joomla.org/joomla-cms/7718</a>.</sub>

**Pull request merged**